.rubocop.yml

```diff
@@ -1,5 +1,5 @@
 Metrics/LineLength:
-  Max: 120
+  Max: 80
```

Comment:
I'm personally a fan of having slightly longer lines, we don't all have 800x600 screens any more!